doc/classes/Viewport.xml

```diff
@@ -6,7 +6,7 @@
 	<description>
 		A Viewport creates a different view into the screen, or a sub-view inside another viewport. Children 2D Nodes will display on it, and children Camera 3D nodes will render on it too.
 		Optionally, a viewport can have its own 2D or 3D world, so they don't share what they draw with other viewports.
-		If a viewport is a child of a [ViewportContainer], it will automatically take up its size, otherwise it must be set manually.
+		If a viewport is a child of a [ViewportContainer], the viewport will automatically take up the ViewportContainer's size, otherwise it must be set manually.
```

Comment:
```suggestion
		If a viewport is a child of a [ViewportContainer], the viewport will automatically take up the container's  size, otherwise it must be set manually.
```

We usually link the class name by using `[ViewportContainer]`. But here I think it's better to avoid being repetitive.